umap/views.py

```diff
@@ -422,7 +422,7 @@ def _urls_for_js(urls=None):
 
 
 def simple_json_response(**kwargs):
-    return HttpResponse(json.dumps(kwargs))
+    return HttpResponse(json.dumps(kwargs), content_type="application/json")
```

Comment:
@yohanboniface I made a PR just for that line, does it look OK to you?

Reply:
It does!